superset-frontend/src/SqlLab/components/RunQueryActionButton.tsx

```diff
@@ -69,6 +69,8 @@ const onClick = (
 const StyledButton = styled.span`
   button {
     line-height: 13px;
+    transition: background-color 0ms;
+    // this is to over ride a previous transition built into the component
```

Comment:
maybe add the comment above the `transition` line? I think that's how we usually comment stuff in the repo

Reply:
changed! Thank you.